app/views/groups/_lms_credentials_modal.html.erb

```diff
@@ -0,0 +1,50 @@
+<div id="lmscredentialsModal-<%= assignment.id %>" class="modal fade" role="dialog">
+  <div class="modal-dialog primary-modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header primary-modal-header">
+        <h4 class="modal-title"><%= assignment.name %></h4>
+        <button type="button" class="close" data-dismiss="modal">&times;</button>
+      </div>
+      <div class="modal-body">
+        <h5>LMS Credentials</h5>
+        <br>
+        <div>
+          <strong>Tool Url:</strong>
+          <p id ="toolurl" class ="groups-showlmscredentialsmodal-links-text"><%= URI.join request.original_url, '/lti/launch' %></p>
```

Comment:
class name is too long, also what was the context for adding original_url ?

Reply:
* I will shorten the class name surely.
* for the ```original_url``` as in my case when I run this via localhost , my lti url becomes ```localhost:3000/lti/launch``` but when in production it should be ```https://circuitverse.org/lti/launch``` so I have used the ```original_url``` such that correct information is shown in the UI irrespective of environment.

Reply:
class name shortened